### PR TITLE
service/dap: log goroutine id and include source info

### PR DESCRIPTION
**Pull request by suzmue**
Log the goroutine id that hit the log message and include the source information. This source information is displayed to the right of the log message and VS Code links it to the appropriate source location.

Updates golang/vscode-go#123

Screen Shot 2021-10-01 at 12 46 34 PM

### CODE REVIEW

**Review by aarzilli — APPROVED** [review]
LGTM

**Review by derekparker — APPROVED** [review]
LGTM

**Review by polinasok — CHANGES_REQUESTED** [review]

**Comment by suzmue** [comment]
Screen Shot 2021-10-05 at 2 08 19 PM

File: service/dap/server.go [hunk 1]
```diff
@@ -3162,18 +3162,21 @@ func (s *Session) handleLogPoints(state *api.DebuggerState) bool {
...
-	// TODO(suzmue): allow evaluate expressions within log points and
-	// consider adding line and goid info to output.
+	// TODO(suzmue): allow evaluate expressions within log points.
 	if msg, ok := bp.UserData.(string); ok {
 		s.send(&dap.OutputEvent{
 			Event: *newEvent("output"),
 			Body: dap.OutputEventBody{
 				Category: "stdout",
-				Output:   msg + "\n",
+				Output:   fmt.Sprintf("> goroutine=%d: %s\n", goid, msg),
```
[the remark this answers is not in the document]
Comment (suzmue) [reply service/dap/server.go]: Done.

**Review by polinasok — APPROVED** [review]

**Pull request merged**